roles/container-engine/containerd/defaults/main.yml

```diff
@@ -73,3 +73,6 @@ containerd_limit_proc_num: "infinity"
 containerd_limit_core: "infinity"
 containerd_limit_open_file_num: "infinity"
 containerd_limit_mem_lock: "infinity"
+
+# If enabled it will user config_path and disable use mirrors config
+# use_config_path: true
```

Comment:
`containerd_use_config_path` maybe better. And default value is false can be more suitable for the container 1.x